datafusion/core/src/datasource/file_format/csv.rs

```diff
@@ -984,12 +983,10 @@ mod tests {
     async fn query_compress_data(
         file_compression_type: FileCompressionType,
     ) -> Result<()> {
-        let runtime = Arc::new(RuntimeEnvBuilder::new().build()?);
```

Comment:
This simply creates a default RuntimeEnv which the SessionStateBuilder already does, so there is no need to do it again